packages/cli/src/commands/generate/service/__tests__/scenario.test.js

```diff
@@ -141,6 +144,21 @@ describe('the scenario generator', () => {
     expect(typeof value).toBe('number')
   })
 
+  test('scenarioFieldValue returns a byte for Byte types NodeJS Buffer type', () => {
+    const field = { type: 'Bytes' }
+    const value = service.scenarioFieldValue(field)
+    // GraphQLByte.parseValue(encodedValue) examples https://github.com/Urigo/graphql-scalars/blob/aeb8aebc9738d533e0cac4cf6ddbad7d1555814e/tests/Byte.test.ts
+    // we have to add a new test to check if tis a Buffer
+    if(Buffer.isBuffer(value)) {
+      expect(typeof value).toBe('object')
+    } else {
+      console.log("Value failed isBuffer")
+      process.exit(1)
+    }
+    expect(value).toEqual(GraphQLByte.parseValue(value))
```

Comment:
Is it possible to parse this value (or verify the contents without importing a library?

Reply:
I'll look @dac09 but when I previously tried to implement Bytes last year, I got held up in the scenarios and tests when trying to transform the Bytes into some value that could be set in the scenario.

This parsing approach for `GraphQLByte` is part of the `graphql-scalars` test package, or so they found, and we though it might solve them problems. But -- will find out.